src/lib/custom-animations/ScaleOnPressIn.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {Pressable, PressableProps} from 'react-native'
+import Animated, {
+  cancelAnimation,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated'
+
+export function ScaleOnPressIn({
```

Comment:
Can we call it `TouchableScale` or `PressableScale` or something so it's clear it's a type of Touchable

Reply:
yerp